enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -181,16 +238,19 @@ async def start_kernel(self, *args, **kwargs):
                 kernel_name=kwargs["kernel_name"], username=username
             )
         )
-
         # Check max kernel limits
         self._enforce_kernel_limits(username)
 
         RemoteMappingKernelManager.pending_requests.increment(username)
+        tenant_id = kwargs.pop(
```

Comment:
can we read this from `kwargs['env']` ?

Reply:
Yes.  Per above we'll use that as our form of conveyance.